.github/workflows/run-cluster.yaml

```diff
@@ -45,12 +41,20 @@ on:
         default: ""
 
 jobs:
+  build-commit:
+    uses: ./.github/workflows/build-commit.yaml
+    with:
+      arch: x86
+      python_version: ${{ inputs.python_version }}
+    secrets:
+      ACTIONS_AWS_ROLE_ARN: ${{ secrets.ACTIONS_AWS_ROLE_ARN }}
```

Comment:
Should this `job` not be optionally executed depending on whether:

1. User wants a specific published Daft version vs a ref
2. In the latter case, if a previously built wheel can be found

Reply:
Yes, that was my initial approach. However, I wanted to just get something basic up for this PR.

Possible implementation:
- we have a `check-if-built` job that runs first
  - if it does, it:
    - sets `wheel_url` to whatever it found
    - sets `should_build` to false
  - if not, then:
    - sets `should_build` to true
- the `build` job only execute if if `${{ needs.check-if-built.outputs.should_build == 'true' }}`

The problem that I was anticipating is how do I consume the wheel_url in the steps in `run-cluster`? Right now, it's `${{ needs.build-commit.outputs.wheel_url }}`, but if that step doesn't execute, then how does it find the wheel_url?

Possibly `${{ needs.build-commit.outputs.wheel_url || needs.check-if-built.outputs.wheel_url }}`? Will need to look into that some more.

Reply:
Used the `if: always()`. Works well.